src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -912,6 +914,65 @@ OverlayImpl::crawl()
     return jv;
 }
 
+Json::Value
+OverlayImpl::getServerInfo()
+{
+    bool const humanReadable = false;
+    bool const admin = false;
+    bool const counters = false;
+
+    Json::Value server_info = app_.getOPs().getServerInfo(humanReadable, admin, counters);
+
+    // Filter out some information
+    server_info.removeMember(jss::hostid);
+    server_info.removeMember(jss::load_factor_fee_escalation);
+    server_info.removeMember(jss::load_factor_fee_queue);
+
+    if (server_info.isMember(jss::validated_ledger))
+    {
+        Json::Value& validated_ledger = server_info[jss::validated_ledger];
+
+        validated_ledger.removeMember(jss::base_fee);
```

Comment:
Why do we remove these three members?

Reply:
That was by request of dgaray, he mentioned they weren't very useful/just noise.